symplyphysics/definitions/volume_flow_rate.py

```diff
@@ -0,0 +1,40 @@
+from sympy import (Eq, Derivative, dsolve)
+from symplyphysics import (units, Quantity, Function, Symbol, print_expression, validate_input,
+    validate_output)
+from symplyphysics.core.expr_comparisons import expr_equals
+from symplyphysics.definitions import mass_flow_rate as mass_flow_rate_law
+from symplyphysics.definitions import density_from_mass_volume as density_law
+
+# Description
+## The rate of change in the volume of matter. For example, the outflow of a substance
```

Comment:
Do we really need this law? Is there any case when we cannot use mass flow rate?